src/styles/base/_typography.scss

```
@@ -19,6 +19,12 @@ html {
     font: var(--typography-html-font, $body-font-size $font-content);
   }
   quotes: "“" "”";
+
+  @each $lang, $quotes in $localizedQuotes {
+    :lang(#{$lang}) {
```

We apply the lang on the `html` right?
```suggestion
    &:lang(#{$lang}) {
```